### DISCUSSION TITLE
AL Support in Copilot Labs VSCode extension

### DISCUSSION THREAD

**Discussion by bkmarcotte-wei** [community/community#55858] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Would be great if the Copilot Labs VSCode extension had support for AL (Microsoft Business Central development language)

**Comment by JesusCastroFernandez** [comment]
Indeed, it would be beneficial if the Copilot Labs VSCode extension had support for AL, the development language used in Microsoft Business Central. AL is specifically designed for customizations and extensions within the Dynamics 365 ecosystem. Having AL support in the Copilot Labs extension would likely enhance the development experience for Microsoft Business Central developers using Visual Studio Code.
